board/safety/safety_toyota.h

```diff
@@ -203,6 +226,19 @@ static bool toyota_tx_hook(const CANPacket_t *to_send) {
       }
     }
 
+    // SecOC cars block any form of LTA actuation for now
+    if (toyota_secoc_car && (addr == 0x131)) {
+      bool lta_request = GET_BIT(to_send, 3U);
+      bool lta_request2 = GET_BIT(to_send, 0U);
+      int lta_angle_msb = GET_BYTE(to_send, 2);
+      int lta_angle_lsb = GET_BYTE(to_send, 3);
+
+      bool actuation = lta_request || lta_request2 || (lta_angle_msb != 0) || (lta_angle_lsb != 0);
+      if (actuation) {
+        tx = false;
```

Comment:
@maxime-desroches Can you help me understand why the new mutation test dislikes this?

https://github.com/commaai/panda/actions/runs/11135924813/job/30946827139?pr=2042

Reply:
Should be fixed with a rebase